wgpu-core/src/command/clear.rs

```diff
@@ -5,24 +5,25 @@ use crate::device::trace::Command as TraceCommand;
 use crate::{
     align_to,
     command::CommandBuffer,
+    device::Device,
     get_lowest_common_denom,
-    hub::{Global, GlobalIdentityHandlerFactory, HalApi, Token},
-    id::{BufferId, CommandEncoderId, DeviceId, TextureId},
-    init_tracker::MemoryInitKind,
-    track::TextureSelector,
+    hub::{Global, GlobalIdentityHandlerFactory, HalApi, Resource, Token},
+    id::{self, BufferId, CommandEncoderId, DeviceId, TextureId},
```

Comment:
nit: if you are including `id`, then just write full path to the ids, i.e .`id::BufferId` everywhere.
this is how I was writing the rest of the code anyway. When there is only 1 or 2 things included, may include them directly. If more - just use the submodule qualified path.

Reply:
reason `self` gets into the mix is `id::Valid()`. Wouldn't mind replacing all the ids here with `id::ThingId`, but that would make this diff a bit messy